CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 Changes by Version
 ==================
 
-2.0.1 (unreleased)
+2.1.0 (2016-02-17)
 -------------------
 
-- No changes yet
+- Support https://github.com/uber-go/zap
```

Comment:
* Add adapter for `zap.Logger` (https://github.com/uber-go/zap)
* Move logging API to `./log/` package